lib/server.js

```diff
@@ -4,6 +4,29 @@ const Hapi = require('hapi');
 const registrationMan = require('./registerPlugins');
 
 /**
+ * If we're throwing errors, let's have them say a little more than just 500
+ * @method prettyPrintErrors
+ * @param  {Hapi.Request}    request Hapi Request object
+ * @param  {Hapi.Reply}      reply   Hapi Reply object
+ */
+function prettyPrintErrors(request, reply) {
+    if (request.response.isBoom) {
+        const err = request.response;
+        const errName = err.output.payload.error;
+        const errMessage = err.message;
```

Comment:
Boom will automatically hide all 500 errors from the end user.  https://github.com/hapijs/boom/blob/master/lib/index.js#L136-L138

This reverts that feature.